infrastructure/transform/main.tf

```diff
@@ -11,7 +11,6 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-
```

Comment:
We probably do want the blank line here so there's a space under the file header like with the other files in the repo